mix.exs

```diff
@@ -67,7 +67,7 @@ defmodule Preview.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
-      setup: ["deps.get", "run priv/seeds.exs", "cmd npm install --prefix assets"]
+      setup: ["deps.get", "run priv/seeds.exs", "cmd cd assets && yarn install"]
```

Comment:
let's use the same thing as on https://github.com/hexpm/hexpm/blob/3608a9aa16d321f8f7d0890528676c0821d85686/mix.exs#L80:

```suggestion
      setup: ["deps.get", "run priv/seeds.exs", "cmd yarn install --cwd assets"]
```